### PR TITLE
Add Configuration Flag To Disable Sending Customer Data To Nosto

### PR DESCRIPTION
**Pull request by supercid**

### CODE REVIEW

File: app/code/community/Nosto/Tagging/Block/Customer.php [hunk 1]
```diff
@@ -76,15 +76,21 @@ public function getVisitorChecksum()
...
-     * @return Nosto_Object_Customer
+     * @return Nosto_Object_Customer|null
      */
     public function getNostoCustomer()
     {
+        $helper = Mage::helper('nosto_tagging');
+        $store = Mage::app()->getStore();
+        $nostoCustomer = new Nosto_Object_Customer();
+        /* @var Nosto_Tagging_Helper_Data $helper */
+        if ($helper->getSendCustomerData($store)) {
```
Comment (hannunosto) [inline app/code/community/Nosto/Tagging/Block/Customer.php]: Shouldn't this be negated? ` if (!$helper->getSendCustomerData($store)) {`

Comment (supercid) [reply app/code/community/Nosto/Tagging/Block/Customer.php]: Fixed, thanks

File: app/code/community/Nosto/Tagging/Block/Customer.php [hunk 2]
```diff
@@ -76,15 +76,21 @@ public function getVisitorChecksum()
...
      * Populates Nosto customer object
      *
-     * @return Nosto_Object_Customer
+     * @return Nosto_Object_Customer|null
      */
     public function getNostoCustomer()
     {
+        $helper = Mage::helper('nosto_tagging');
+        $store = Mage::app()->getStore();
+        $nostoCustomer = new Nosto_Object_Customer();
```
Comment (mridang) [inline app/code/community/Nosto/Tagging/Block/Customer.php]: Why is object being instantiated prior the check? This needs to be move after.

Comment (supercid) [reply app/code/community/Nosto/Tagging/Block/Customer.php]: Good point. Fixed, thanks.

**Comment by supercid** [comment]
@mridang @hannunosto The last commit disables the personal user data sending from the order placement if data sending is disabled in the plugin settings.

**Pull request merged**